src/engine/SCons/Action.py

```diff
@@ -485,7 +485,7 @@ def execute(self, target, source, env):
             # DLADD: JRH 20151022 SF37871/SF37872: DLE now contains to many .java and .cs files
             #                                      to compile with the standard windows cmd.exe
             # on windows, use powershell when available/necessary (to avoid command line length limit failures)
-            if (sys.platform == 'win32') and (cmd_line[0] == 'javac'): # limit the patch to javac, for now
+            if (sys.platform == 'win32'): # limit the patch to javac, for now
```

Comment:
You should probably change this comment as well.  I advised JR against this clause when I reviewed this initially, but he talked me out of it.